### PR TITLE
Adds support for automating release version numbers throughout the project and sets everything to 1.0.0-dev

### PR DESCRIPTION
**Pull request by balsama**

### CODE REVIEW

File: src/Composer/ReleaseVersion.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+namespace Acquia\headless_lightning\Composer;
+
+use Acquia\Lightning\Composer\ReleaseVersion as BaseReleaseVersion;
+use Composer\Script\Event;
+
+/**
+ * Updates the version number in Headless Lightning's component info files.
+ */
+class ReleaseVersion extends BaseReleaseVersion {
```
Comment (phenaproxima) [inline src/Composer/ReleaseVersion.php]: We don't need this class.

execute() doesn't do anything but call the parent method, so it's not necessary; and updateMakeFile() acts on any build-lightning.make found in the profile directory. If none are found, it effectively does nothing. So we can remove this class, and our composer.json can just call the script that comes with Lightning itself.

File: headless_lightning.info.yml [hunk 2]
```diff
@@ -1,33 +1,18 @@
...
-    - history
-    - rdf
-    - menu_ui
-    - metatag
-    - lightning_layout
-    - lightning_contact_form
-    - lightning_search
-    - lightning_landing_page
-    - views_ui
+  excluded_dependencies: [block_content, breakpoint, contextual, quickedit, history, rdf, menu_ui, metatag, lightning_layout, lightning_contact_form, lightning_search, lightning_landing_page, views_ui]
```
Comment (phenaproxima) [inline headless_lightning.info.yml]: Ugh. I don't like this, but there's not a whole lot we can do about it right now anyway. So I guess I'll let it slide.

File: src/Composer/ReleaseVersion.php (hunk 1 above)
Comment (balsama) [inline src/Composer/ReleaseVersion.php]: doh! Indeed.

**Comment by phenaproxima** [comment]
This looks good. Cut and print!

**Pull request merged**